hc/settings.py

```diff
@@ -20,10 +20,14 @@
 HOST = "localhost"
 SECRET_KEY = "---"
 DEBUG = True
-ALLOWED_HOSTS = []
-DEFAULT_FROM_EMAIL = 'healthchecks@example.org'
+# ALLOWED_HOSTS = []
+# DEFAULT_FROM_EMAIL = 'healthchecks@example.org'
 USE_PAYMENTS = False
-
+EMAIL_HOST = "smtp.gmail.com"
```

Comment:
I did this in order to get an actual email in my mailbox. I was using my gmail account but I'll change this back to the default settings.